core/arch/arm/include/tee/entry_fast.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2015, Linaro Limited
+ * SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Prefer to not break up the copyright statement with this.

Reply:
Will fix.